code/modules/mob/living/living_movement.dm

```diff
@@ -41,6 +41,14 @@
 	else
 		remove_movespeed_modifier(MOVESPEED_ID_LIVING_TURF_SPEEDMOD)
 
+/mob/living/proc/update_pull_movespeed()
+	if(pulling && isliving(pulling))

```

Comment:
wouldn't isliving(null) fail? don't think you need both here.

Reply:
I had to think about why I did this for a minute, and it was because I was thinking that just the `if(pulling)` could take advantage of short circuiting by skipping the `istype` check in the `isliving` define, but I'm not sure if it would be any faster now. I've heard `istype` is somewhat expensive but I don't know what its performance is like compared to direct `if` checks.

Reply:
me either lol

Reply:
innocent until proven guilty tbh